AppCenterDistribute/AppCenterDistribute/MSACDistributeUtil.m

```diff
@@ -13,10 +13,17 @@
   static NSBundle *bundle = nil;
   static dispatch_once_t predicate;
   dispatch_once(&predicate, ^{
-    // The resource bundle is part of the main app bundle, e.g. .../Puppet.app/AppCenterDistribute.bundle
-    NSString *mainBundlePath = [[NSBundle bundleForClass:[MSACDistribute class]] resourcePath];
-    NSString *frameworkBundlePath = [mainBundlePath stringByAppendingPathComponent:APP_CENTER_DISTRIBUTE_BUNDLE];
-    bundle = [NSBundle bundleWithPath:frameworkBundlePath];
+  // The resource bundle is part of the main app bundle, e.g. .../Puppet.app/AppCenterDistribute.bundle
```

Comment:
If this macro is defined you should just use that as `bundle`. You should not start from the main bundle. 

This won't work correctly for a dynamic library, and is depending upon an internal implementation detail.